sandbox/src/Home.js

```diff
@@ -8,7 +8,7 @@ function HomeWithHistory({ history }) {
     if (loc.pathname !== previousPath) {
       const instanceName = loc.pathname.includes("orgTwo") ? "organizationTwo" : "alloy";
       window[instanceName]("event", {
-        type: "viewStart",
+        isViewStart: true,
```

Comment:
I was wondering if we should use `viewStart: true`, instead of `isViewStart`. We had a discussion around NOT using options/configs that look like functions or method calls.